src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -25,7 +24,6 @@ public partial class AppDomain : MarshalByRefObject
 
         public static AppDomain CurrentDomain => s_domain;
 
-#if !uapaot
```

Comment:
I did a quick search and it looks like these were the only places where we used the uapaot constant, so now that it's not used anymore, do you mind removing it from the .csproj?